src/workerd/jsg/README.md

```diff
@@ -396,13 +421,13 @@ There are generally three ways of using `jsg::Function`:
 
 For the first item, imagine the following case:
 
-We have a C++ function that takes a callback argument. The callback argument takes a single `int`
+We have a C++ function that takes a callback argument. The callback argument takes an `int`
 as an argument (Ignore the `jsg::Lock& js` part for now, we'll explain that in a bit). As soon
 as we get the callback, we invoke it as a function.
 
 ```cpp
 void doSomething(jsg::Lock& js, jsg::Function<void(int)> callback) {
-  callback(1);
+  callback(js, 1);
```

Comment:
Should the declaration of `callback` be `jsg::Function<void(jsg::Lock&,int)> callback`?

Reply:
No, the `jsg::Lock&` argument is not included in the declaration bit, just required when it is actually called.